interfaces/IEssosSample.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "Module.h"
+
+namespace WPEFramework {
+namespace Exchange {
+
+    struct EXTERNAL IEssosSample : virtual public Core::IUnknown {
+        enum { ID = ID_ESSOS_SAMPLE };
+
+        virtual ~IEssosSample() {}
+        virtual uint32_t Initialize(PluginHost::IShell* service) = 0;
+        virtual void Deinitialize(PluginHost::IShell* service) = 0;
+        virtual uint32_t Configure(PluginHost::IShell* service) = 0;
+        virtual uint32_t setWaylandDisplay(const string &input) = 0;
```

Comment:
This is a bit like having two standards in one interface. In interfaces, we alsways use PascalCase naming to identify methods. Here we are observing camelCase and PascalCase. Let please align it all to PascalCase.
Also Why do we have Initialize and Deinitialize here that take an IShell* I would not expect them here but the would be part of the IPlugin interface, otherwise it might become confusing, what does which Initialize/Deinitialize mean ?...